### PR TITLE
[test_0]:HelloWorld pr for users.

### PR DESCRIPTION
**Pull request by WBSLZF**
### What?
- brief: 测试pr
- author: WBSLZF

### What changes are included in this PR?

本次pr创建并运行了一个HelloWorld程序
### Are these changes tested?
编写并运行了HelloWorld程序
image

### CODE REVIEW

**Comment by Light-City** [comment]
don't modify my pr test and copy my pr.

**Comment by WBSLZF** [comment]
Sorry，I fixed my pr part. Can i get a LGTM? QWQ

**Comment by tigerwe** [comment]
> Sorry，I fixed my pr part. Can i get a LGTM? QWQ

Your commit format is not standardized, please follow https://github.com/datawhalechina/whale-starry/blob/main/stl/rules/commit.md

**Comment by tigerwe** [comment]
> > Sorry，I fixed my pr part. Can i get a LGTM? QWQ
> 
> Your commit format is not standardized, please follow https://github.com/datawhalechina/whale-starry/blob/main/stl/rules/commit.md

and your directory structure is incorrect

File: .DS_Store
Comment (martinx) [inline .DS_Store:1]: Please remove

File: stl/.DS_Store
Comment (martinx) [inline stl/.DS_Store:1]: Please remove

File: stl/img/.DS_Store
Comment (martinx) [inline stl/img/.DS_Store:1]: Please remove

File: stl/fight_flow/light-city/src/main.cpp
Comment (martinx) [inline stl/fight_flow/light-city/src/main.cpp:1]: Please do not delete other guy's codes.

File: stl/fight_flow/README.md
Comment (martinx) [inline stl/fight_flow/README.md:1]: Please do not delete other guy's codes.

**Comment by martinx** [comment]
pr title is wrong

File: stl/.DS_Store
Comment (WBSLZF) [reply stl/.DS_Store:1]: Sorry，I am confused. I delete the .DS_Store file based on the talk in the wetchat group thus this block turn to red.

File: stl/fight_flow/light-city/src/main.cpp
Comment (WBSLZF) [reply stl/fight_flow/light-city/src/main.cpp:1]: Sorry, i still confused. In the begining, i heard that we should make our own helloworld cpp program so i add a new cpp file named the hello_world.cpp based on the fact that some people get LGTM through this code. Then another instructor tells me don't modify the structure which means that i shouldn't add new cpp file in this structure. Later, i modified the code in the raw main.cpp file, then i get this. I confused by the obscure standard.

Comment (WBSLZF) [reply stl/fight_flow/light-city/src/main.cpp:1]: image

**Pull request closed without merging**